src/base64.h

```diff
@@ -52,8 +52,8 @@ extern const int8_t unbase64_table[256];
 
 
 template <typename TypeName>
-size_t base64_decode_slow(char* dst, size_t dstlen,
-                          const TypeName* src, size_t srclen) {
+size_t base64_decode_slow(char* const dst, const size_t dstlen,
+                          const TypeName* const src, const size_t srclen) {
```

Comment:
@richardlau The first `const` declares the pointed-to memory immutable, the second one declares the pointer itself immutable (i.e., not reassignable.)